### ISSUE TITLE
Suggesting to whitelist wl.spotify.com - without it password reset links don't work.

### ISSUE THREAD

**Issue by Marcoevich** [StevenBlack/hosts#1727]
I just tried a password reset on Spotify, but the password reset link, wl.spotify.com, was blocked. I suggest that you whitelist this one.

**Comment by StevenBlack** [comment]
Thank you Rick @Marcoevich!  The `wl.spotify.com` domains comes to us via [Adguard CNAME-cloaked trackers](https://github.com/AdguardTeam/cname-trackers).  See the `readme` there to understand why this is a problem.

Can you post this in [Adguard's issues](https://github.com/AdguardTeam/cname-trackers/issues) and see what they say.

I support whatever Adguard decides to do.  By guess: it stays blocked by Adguard, but you're welcome to try.

Keep us informed?

**Comment by dnmTX** [comment]
We're all aware by now about **spotify** and that's why is off the table here.They shouldn't be messing with it neither.

**Comment by ameshkov** [comment]
Well, as I've said before, CNAME-cloaked trackers list only shows what domains are actually CNAME's to known tracking domains. It's not a blocklist, it's just a list, it's up to the downstream lists to decide what to do if blocking causes issues somewhere. In our blocklists we selectively unblock problematic domains if the issue is important.

**Comment by StevenBlack** [comment]
Thank you Andrey @Marcoevich.

I'm inclined to live with this.  Let me think about it @Marcoevich, let's let this one simmer for awhile before deciding one way of another.

**Comment by dnmTX** [comment]
Steve @StevenBlack what is the situation with implementing own whitelist here. Is there a option for this?
I don't really remember being discussed in the past.

Andrey @ameshkov thanks btw for the quick response 👍

**Comment by StevenBlack** [comment]
Pull requests speak loud and carry weight Dan @dnmTX 🙃

**Comment by dnmTX** [comment]
> Pull requests speak loud and carry weight

Even if the domain in question coming from upstream and it will reappear after the following update?

**Comment by StevenBlack** [comment]
Dan @dnmTX well for starters we're using Andrey @ameshkov list at cross-purposes.  But we get occasional Type I errors for our specific purpose from all our sources, not just Andrey's.  

In our composition, including Adguard CNAME-list is a conscious choice, and that can be revisited.  That said, overall I'm delighted with this list as a source of domains.

I presently imagine a whitelist implementation a potential post-process to cull our amalgamated lists at some point before the optional userland whitelist culling hook fires.

I'm suspicious of implementing a compensatory whitelist here for one simple reason: *where does that end?*  

On balance I trust our curators and that has been wonderful guiding principle since 2012 when this repo's mission was conceived.

I'm leaning towards won't fix for this particular issue.  Maybe that's kicking the can down the road.

**Comment by dnmTX** [comment]
> I'm suspicious of implementing a compensatory whitelist here for one simple reason: where does that end?

I got that and no arguments there,i was only mentioned **whitelist** due to the fact that the main curator of that list emphasised numerous times already that the list is only *informative* and it falls to downstream to do any adjustments.
That's all 👍

**Comment by Marcoevich** [comment]
I see this issue has been marked as stale and will be closed soon if no one replies. @StevenBlack can you decide what to do with this issue? I'm happy either way as long as there's human thought process behind it.

**Comment by StevenBlack** [comment]
Hi Rick @Marcoevich thank you for the reminder.

I won't change this.  The CNAME list occasionally yields false positives, and that's unfortunate.

Fact is in this case, `wl.spotify.com` maps to `thirdparty.bnc.lt` and you know what, that's just sketch.

```
 dig wl.spotify.com

; <<>> DiG 9.10.6 <<>> wl.spotify.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 14955
;; flags: qr rd ra; QUERY: 1, ANSWER: 3, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;wl.spotify.com.			IN	A

;; ANSWER SECTION:
wl.spotify.com.		81	IN	CNAME	thirdparty.bnc.lt.
thirdparty.bnc.lt.	50	IN	A	50.18.205.147
thirdparty.bnc.lt.	50	IN	A	52.9.144.63

;; Query time: 7 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Wed Dec 08 11:57:40 EST 2021
;; MSG SIZE  rcvd: 106
```

Closing.

**Comment by rube200** [comment]
Apparently this is still a thing it also happen when you ask to reset the password
